resources/assets/coffee/_classes/nav.coffee

```diff
@@ -64,6 +64,21 @@ class @Nav
     @_data[0].dataset
 
 
+  delayedHidePopup: =>
+    return if @currentMode() == 'user' && !currentUser.id?
+    return if @currentMode() == 'search'
+
+    Timeout.clear @hideTimeout
+    Timeout.clear @showTimeout
+    @hideTimeout = Timeout.set 250, @hidePopup
+
+
+  delayedShowPopup: =>
+    Timeout.clear @hideTimeout
+    Timeout.clear @showTimeout
+    @showTimeout = Timeout.set 250, @showPopup
```

Comment:
Looks like 100 is a bit too low to allow passing header vertically without showing the popup.